text/0000-bee-model-crate/0000-bee-model-crate.md

```diff
@@ -0,0 +1,334 @@
++ Feature Name: `bee-model-crate`
```

Comment:
We should rename it to something like `iri-transaction-model` or `mainnet-transaction-model` or `v1-transaction-model`.

Reply:
I'd vote for versioning since `mainnet-transaction-model` is not time-proof and `iri-transaction-model` is not really the case.

Reply:
How do you want to version it?